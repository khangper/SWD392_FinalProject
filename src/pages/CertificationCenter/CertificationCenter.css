.CC-grid-container{
    display: grid;
  grid-template-columns: 1fr; 

}
  .CC_215certibg {
    float: left;
    width: 100%;
    padding: 80px 0;
    background: #333;
    position: relative;
}

.CC_215certibg:before {
    content: "";
    
    left: 0;
    right: 0;
    top: 0;
    bottom: 0;
    position: absolute;
    pointer-events: none;
    background-size: cover;
    opacity: 0.1;
}


.CC-cert_banner_text {
	text-align: center;
}

.CC-cert_banner_text h1 {
	font-size: 60px;
    font-weight: 600;
	font-family: 'Roboto', sans-serif;
    margin-bottom: 20px;
	text-align: center;
	color: #fff;
}

.CC-cert_banner_text p {
	font-size: 24px;
    margin-bottom: 0;
    font-weight: 400;
	font-family: 'Roboto', sans-serif;
	text-align: center;
	color: #fff;
}

.CC-certi_icons {
	padding-top: 56px;
}


.CC-certi_icons li {
	display: inline-block;
}

.CC-edttslogo img {
    width: 135px;
}

.CC-edttslogo1 img {
    width: 135px;
}
.CC-edttsplus img {
    width: 80px;
    margin: 24px 40px;
}
.CC-certi-btn {
    margin-top: 80px;
    border: 0;
    height: 50px;
    text-align: center;
    padding: 0 100px;
    font-size: 14px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #fff;
    border-radius: 3px;
    background: #ed2a26;
}
.CC-certi-btn:hover{
    cursor: pointer;
}

.CC_215xd5 {
    padding: 62px 0;
    float: left;
    width: 100%;
	background: #fff;
}
.CC-title889 h2 {
    font-size: 24px;
    font-weight: 600;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 7px !important;
    text-align: center;
    color: #333;
    line-height: 26px;
}
.CC1-grid-container {
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
    gap: 1rem;
    max-width: 1061px;
    margin: 0 auto;
 }
.CC1-grid-item {
    display: flex;
    flex-direction: column;
    align-items: center; /* Center horizontally */
    justify-content: center; /* Center vertically */
    text-align: center; /* Center text if needed */
    padding: 1rem; /* Optional: Add padding to each grid item */
}
.ui[class*="left icon"].swdh19.input>input {
    padding-left: 1em !important;
    padding-right: 2em !important;
    height: 40px !important;
    font-size: 14px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
}
.CC-login-btn {
    font-size: 14px;
    margin-top: -21px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #fff;
    background: #ed2a26;
    border-radius: 3px;
    text-align: center;
    border: 0;
    width: 100%;
    height: 40px;
}
.CC-login-btn:hover{
    background-color: #333;
    cursor: pointer;
}
.CC-title589 h2 {
	font-size: 24px;
    font-weight: 600;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 17px !important;
	text-align: center;
	color: #333;
	line-height: 26px;
}
.CC-title589 p {
	font-size: 16px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 0 !important;
	text-align: center;
	color: #686f7a;
	line-height: 26px;
	padding: 0 300px;
}
.CC-tabs {
    list-style-type: none;
    padding: 0;
    margin: 0;
    display: flex;
    justify-content: center;

}
.CC-tabs li {
    cursor: pointer;
    padding: 10px;
    border-radius: 3px;
    margin: 0 10px;
    border-bottom: 2px solid transparent;
    transition: border-bottom-color 0.3s ease;
    font-weight: 600;
}
.CC-tabs li:hover {
    color: white;
    background-color: red;
    padding: 10px;
    border-radius: 3px;
    font-weight: 600;
}
.CC-tabs li.active {
    color: white;
    background-color: red;
    padding: 10px;
    border-radius: 3px;
    font-weight: 600;
}
.tab-content {
    margin-top: 69px;
}
.CC-container {
    padding-top: 20px;
    padding-bottom: 20px;
    transition: padding-left 0.4s;
    overflow-y: auto;
    flex: 1 1;
}
.CC-certificates {
    display: flex;
    justify-content: center;
  }
  
  .CC-certificates .CC3-grid-container {
    display: grid;
    grid-template-columns: repeat(4, 1fr);
    gap: 80px;
  }
  

  .CC-certilink__152 {
    background: #fff;
    border-radius: 3px;
    border: 1px solid #efefef;
    padding: 20px;
    box-shadow: none;
    flex: 1;
    float: left;
    color: #333 !important;
    width: 100%;
    font-size: 14px;
    line-height: 26px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    margin-top: -34px;
    text-align: center;
}

.CC-certilink__152:hover {
    color: #fff !important;
	background: #ed2a26 ;
}


.CC_485td5 {
    padding: 62px 0;
    float: left;
    width: 100%;
	position: relative;
	background: #333;
}

.CC_485td5:before {
    content: "";
   
    left: 0;
    right: 0;
    top: 0;
    bottom: 0;
    position: absolute;
    pointer-events: none;
    background-size: cover;
    opacity: 0.06;
}

.CC-titleceti89 h2 {
    font-size: 24px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 17px !important;
    text-align: center;
    color: #fff;
    line-height: 26px;
}


.CC4-grid-container {
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
    gap: 5rem;
    justify-content: center;
    align-items: center;
    margin: 0 auto;
    width: max-content;
    display: flex;
}

.CC4-grid-item {
    display: flex;
    flex-direction: column;
    align-items: center; /* Center items within each grid item */
    text-align: center;
}


  .who_get {
    text-align: center;
    margin-top: 43px;
}

.who_img {
    width: 150px;
    height: 150px;
    background: #fff;
    border-radius: 100%;
    display: inline-block;
	padding: 23px 0;
	border: 5px solid #fff;
	box-shadow: inset 0px 0px 5px 2px rgba(0, 0, 0, 0.1);
}

.who_get img {
    width: 90px;
}
.who_get h4 {
    font-size: 18px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #fff;
    margin-top: 27px;
    text-align: center;
    line-height: 26px;
}
.CC-title589 h2 {
	font-size: 24px;
    font-weight: 600;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 17px !important;
	text-align: center;
	color: #333;
	line-height: 26px;
}
.CC-title589 p {
	font-size: 16px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 0 !important;
	text-align: center;
	color: #686f7a;
	line-height: 26px;
	padding: 0 300px;
}
.CC5-grid-container {
    display: grid;
    grid-template-columns: 1fr; /* Adjust the columns as needed */
    gap: 20px; /* Adjust the gap between grid items as needed */
    padding: 0 180px;
}
.CC-knowledge_dts {
	text-align: center;
	margin-top: 30px;
}

.CC-knowledge_dts p {
	font-size: 14px;
    font-weight: 500;
    font-family: 'Open Sans', sans-serif;
    line-height: 26px;
    color: #686f7a;
    text-align: center;
}
.CC-knowledge_btn {
    margin-top: 23px;
    border: 0;
    height: 50px;
    text-align: center;
    padding: 0 100px;
    font-size: 14px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #fff;
    border-radius: 3px;
    background: #ed2a26;
}

.CC-knowledge_btn:hover {
    color: #fff;
    background: #333;
}
.CC6-grid-container {
    display: grid;
    grid-template-columns: 1fr;
    gap: 20px;
    padding: 0px 1px;
}