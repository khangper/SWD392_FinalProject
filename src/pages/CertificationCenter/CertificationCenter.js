import React, { useState } from 'react'
import './/CertificationCenter.css'
import Header from '../../components/Header-paidmember/Header'
import { Link } from 'react-router-dom'
import Footer from '../../components/Footer/Footer'


function CertificationCenter() {

  const [activeTab, setActiveTab] = useState('development'); // State để lưu tab đang được chọn

    const handleTabClick = (tabId) => {
        setActiveTab(tabId); // Cập nhật state khi người dùng chọn tab
    };
  return (
    <div className="CertificationCenterPage">
      <Header/>

      <div className="CC_215certibg">
             <div className="CC-grid-container">						
						      <div className="CC-cert_banner_text CC-grid-item">						
							 <h1>Certification Center</h1>
							 <p>For Students and Instructors</p>
							 <ul className="CC-certi_icons">
								<li><a href="#" className="CC-edttslogo"><img src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg" alt=""/></a></li>
								<li><div className="CC-edttsplus"><img src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/plus.svg" alt=""/></div></li>
								<li><a href="#" className="CC-edttslogo1"><img src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/certicon.svg" alt=""/></a></li>
							 </ul>
							 <Link to="/certificationfillform">
               <button className="CC-certi-btn">Start Certification</button>
               </Link>
						       </div>
					    </div>

      </div>

      <div className="CC_215xd5">
            <div className="CC-grid-container">
							<div className="CC-title889">
								<h2>Find Certificate</h2>
							</div>												
						</div>


            <div className="CC1-grid-container">
    <div className="CC1-grid-item">
        <div className="ui search focus">
            <div className="ui left icon input swdh11 swdh19">
                <input className="prompt srch_explore" type="text" name="cerificate[number]"  placeholder="# Number"/>
            </div>
        </div>
    </div>
    <div className="CC1-grid-item">
        <div className="ui search focus">
            <div className="ui left icon input swdh11 swdh19">
                <input className="prompt srch_explore" type="text" name="fullname" placeholder="Full Name"/>
            </div>
        </div>
    </div>
    <div className="CC1-grid-item">
      <div className="ui search focus">
      <select className="ui hj145 dropdown cntry152 prompt srch_explore">
            <option value="">Select Category</option>
            <option value="1">Development</option>
            <option value="2">Finance & Accounting</option>
            <option value="3">Design</option>
            <option value="4">Marketing</option>
            <option value="5">Teaching & Academics</option>
        </select>
      </div>

    </div>
    <div className="CC1-grid-item">
      <div className="ui search focus">
      <button className="CC-login-btn" type="submit">Find Certificate</button>
      </div>
        
    </div>
            </div>


      

      </div>


      <div className="CC_215xd5">
             <div className="CC-grid-container">           
						<div className="CC-title589 ">
							<h2>Our Certification</h2>
							<p>We prepared tests for the most popular categories and get cerificate</p>
						</div>
					   </div>

             <div className="CC-grid-container">
           <div className="CC-container">
            <ul className="CC-tabs">
                <li className={activeTab === 'development' ? 'active' : ''} onClick={() => handleTabClick('development')}>
                    Development
                </li>
                <li className={activeTab === 'accounting' ? 'active' : ''} onClick={() => handleTabClick('accounting')}>
                    Finance & Accounting
                </li>
                <li className={activeTab === 'design' ? 'active' : ''} onClick={() => handleTabClick('design')}>
                    Design
                </li>
                <li className={activeTab === 'marketing' ? 'active' : ''} onClick={() => handleTabClick('marketing')}>
                    Marketing
                </li>
                <li className={activeTab === 'teaching' ? 'active' : ''} onClick={() => handleTabClick('teaching')}>
                    Teaching & Academics
                </li>
            </ul>
            <div className="tab-content">
              <div className="CC-grid-container">
              {activeTab === 'development' && 
                  <div  className="CC-development">
                <div className="CC-certificates">
  <div className="CC3-grid-container">
    <a href="#" className="CC-certilink__152">WordPress</a>
    <a href="#" className="CC-certilink__152">HTML CSS</a>
    <a href="#" className="CC-certilink__152">MotoCMS 3</a>
    <a href="#" className="CC-certilink__152">Joomla</a>
    <a href="#" className="CC-certilink__152">OpenCart</a>
    <a href="#" className="CC-certilink__152">Joomla Pro</a>
    <a href="#" className="CC-certilink__152">WordPress Pro</a>
    <a href="#" className="CC-certilink__152">WordPress Elementor</a>
    <a href="#" className="CC-certilink__152">WordPress Elementor Pro</a>
    <a href="#" className="CC-certilink__152">PrestaShop</a>
  </div>
                </div>

                  
                  </div>}
                {activeTab === 'accounting' && 
                    <div className="accounting">
                <div className="CC-certificates">
  <div className="CC3-grid-container">
    <a href="#" className="CC-certilink__152">Accounting</a>
    <a href="#" className="CC-certilink__152">Finance Fundamentabls</a>
    <a href="#" className="CC-certilink__152">Bookkeeping</a>
    <a href="#" className="CC-certilink__152">Political Science</a>
    <a href="#" className="CC-certilink__152">Finace</a>
    <a href="#" className="CC-certilink__152">Cryptocurrency</a>

  </div>
                </div>
                  </div>}
                {activeTab === 'design' && 
                <div className="design">
                                  <div className="CC-certificates">
  <div className="CC3-grid-container">
    <a href="#" className="CC-certilink__152">Adobe Photoshop</a>
    <a href="#" className="CC-certilink__152">Adobe Illustrator</a>
    <a href="#" className="CC-certilink__152">Adobe After Efftects</a>
    <a href="#" className="CC-certilink__152">Adobe InDesign</a>
    <a href="#" className="CC-certilink__152">Unity</a>
    <a href="#" className="CC-certilink__152">Drawing</a>
    <a href="#" className="CC-certilink__152">Game Development Fundamentals</a>
    <a href="#" className="CC-certilink__152">3D Modeling</a>
    <a href="#" className="CC-certilink__152">Motion Graphics</a>
    <a href="#" className="CC-certilink__152">2D Animation</a>
    <a href="#" className="CC-certilink__152">T-Shirt Design</a>



  </div>
                </div>
                </div>}
                {activeTab === 'marketing' && 
                <div>
    <div className="CC-certificates">
      <div className="CC3-grid-container">
         <a href="#" className="CC-certilink__152">Googgle Ads (Adwords)</a>
         <a href="#" className="CC-certilink__152">Google Ads (Adwords) Certification</a>
         <a href="#" className="CC-certilink__152">Socical Marketing</a>
         <a href="#" className="CC-certilink__152">Email Marketing</a>
         <a href="#" className="CC-certilink__152">Business Strategy</a>
         <a href="#" className="CC-certilink__152">SEO</a>
        <a href="#" className="CC-certilink__152">PPC Advertising</a>
        <a href="#" className="CC-certilink__152">Blogging</a>
      </div>
     </div>
                  </div>}
                {activeTab === 'teaching' && <div>
                  <div className="CC-certificates">
      <div className="CC3-grid-container">
         <a href="#" className="CC-certilink__152">Math</a>
         <a href="#" className="CC-certilink__152">Humanities</a>
         <a href="#" className="CC-certilink__152">Engineering</a>
         <a href="#" className="CC-certilink__152">Science</a>
         <a href="#" className="CC-certilink__152">Social Science</a>
         <a href="#" className="CC-certilink__152">English Language</a>
        <a href="#" className="CC-certilink__152">German Language</a>
        <a href="#" className="CC-certilink__152">Sign Language</a>
        <a href="#" className="CC-certilink__152">IELTS</a>
        <a href="#" className="CC-certilink__152">French Language</a>
        <a href="#" className="CC-certilink__152">Spanish Language</a>
        <a href="#" className="CC-certilink__152">English Language</a>

      </div>
     </div>
                  
                  
                  
                  </div>}

              </div>
               
            </div>
        </div>

             </div>
      </div>


       <div className="CC_485td5">
        <div className="CC-grid-container">
						<div className="CC-titleceti89">
							<h2>Who Can Get Benefit From This?</h2>
						</div>
        </div>
        <div className="CC4-grid-container">
    <div className="CC4-grid-item">
      <div className="who_get">
        <div className="who_img">
          <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/student.svg" alt="Student"/>
        </div>
        <h4>Students</h4>
      </div>
    </div>
    <div className="CC4-grid-item">
      <div className="who_get">
        <div className="who_img">
          <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/instructor.svg" alt="Instructor"/>
        </div>
        <h4>Instructor</h4>
      </div>
    </div>
       </div>
       </div>


       <div className="CC_215xd5">
        <div className="CC-grid-container">
        <div className="CC-title589">
							<h2>What Will You Get?</h2>
							<p>Cursus company, which confirms your skills and knowledge of Certification</p>
						</div>
        </div>

        <div className="CC5-grid-container">
    <div className="CC-knowledge_dts">
        <p>Morbi eget elit eget turpis varius mollis eget vel massa. Donec porttitor, sapien eget commodo vulputate, erat felis aliquam dolor, non condimentum libero dolor vel ipsum. Sed porttitor nisi eget nulla ullamcorper eleifend. Fusce tristique sapien nisi, vel feugiat neque luctus sit amet. Quisque consequat quis turpis in mattis. Maecenas eget mollis nisl. Cras porta dapibus est, quis malesuada ex iaculis at. Vestibulum egestas tortor in urna tempor, in fermentum lectus bibendum. In leo leo, bibendum at pharetra at, tincidunt in nulla. In vel malesuada nulla, sed tincidunt neque. Phasellus at massa vel sem aliquet sodales non in magna. Ut tempus ipsum sagittis neque cursus euismod. Vivamus luctus elementum tortor, ac aliquet dolor vehicula et. Nulla vehicula pharetra lacus ornare gravida. Vivamus mollis ullamcorper dui quis gravida. Aenean pulvinar pulvinar arcu a suscipit.</p>
        <button className="CC-knowledge_btn">Knowledge Base</button>
    </div>
        </div>
       </div>


       <div className="CC6-grid-container">
        <Footer/>
       </div>
    </div>
  )
}

export default CertificationCenter
