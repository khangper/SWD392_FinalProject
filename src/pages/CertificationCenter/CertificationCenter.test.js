import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";
import CertificationCenter from "./CertificationCenter";

describe("CertificationCenter", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });

  it("renders the CertificationCenter component", () => {
    const { container } = renderWithProviders(<CertificationCenter />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
  it("validates full name input correctly", () => {
    renderWithProviders(<CertificationCenter />);

    const fullNameInput = screen.getByPlaceholderText("Full Name");
    const errorMessage =
      "Full Name cannot contain special characters or numbers.";

    // Valid input
    fireEvent.change(fullNameInput, { target: { value: "John Doe" } });
    expect(fullNameInput.value).toBe("John Doe");
    expect(screen.queryByText(errorMessage)).not.toBeInTheDocument();
  });

  it("validates certificate number input correctly", () => {
    renderWithProviders(<CertificationCenter />);

    const certificateNumberInput = screen.getByPlaceholderText("# Number");

    // Valid input
    fireEvent.change(certificateNumberInput, { target: { value: "123456" } });
    expect(certificateNumberInput.value).toBe("123456");

    // Invalid input (non-numeric value should not change the value)
    fireEvent.change(certificateNumberInput, { target: { value: "123abc" } });
    expect(certificateNumberInput.value).toBe("123456"); // Value should remain unchanged
  });

  it("switches between tabs correctly", () => {
    renderWithProviders(<CertificationCenter />);

    // Debug the rendered output
    screen.debug();

    // Select tabs using getByRole and getByText
    const developmentTab = screen
      .getAllByText("Development")
      .find((el) => el.tagName === "LI");
    const accountingTab = screen.getByText(
      (content, element) =>
        element.tagName === "LI" && content === "Finance & Accounting"
    );
    const designTab = screen.getByText(
      (content, element) => element.tagName === "LI" && content === "Design"
    );
    const marketingTab = screen.getByText(
      (content, element) => element.tagName === "LI" && content === "Marketing"
    );
    const teachingTab = screen.getByText(
      (content, element) =>
        element.tagName === "LI" && content === "Teaching & Academics"
    );

    // Initially, "Development" tab is active
    expect(developmentTab).toHaveClass("active");

    // Click on "Finance & Accounting" tab
    fireEvent.click(accountingTab);
    expect(accountingTab).toHaveClass("active");
    expect(developmentTab).not.toHaveClass("active");

    // Click on "Design" tab
    fireEvent.click(designTab);
    expect(designTab).toHaveClass("active");
    expect(accountingTab).not.toHaveClass("active");

    // Click on "Marketing" tab
    fireEvent.click(marketingTab);
    expect(marketingTab).toHaveClass("active");
    expect(designTab).not.toHaveClass("active");

    // Click on "Teaching & Academics" tab
    fireEvent.click(teachingTab);
    expect(teachingTab).toHaveClass("active");
    expect(marketingTab).not.toHaveClass("active");
  });

  it("displays the correct content when switching tabs", () => {
    renderWithProviders(<CertificationCenter />);

    // Debug the rendered output
    screen.debug();

    // Select tabs using getByRole and getByText
    const developmentTab = screen
      .getAllByText("Development")
      .find((el) => el.tagName === "LI");
    const accountingTab = screen.getByText(
      (content, element) =>
        element.tagName === "LI" && content === "Finance & Accounting"
    );
    const designTab = screen.getByText(
      (content, element) => element.tagName === "LI" && content === "Design"
    );
    const marketingTab = screen.getByText(
      (content, element) => element.tagName === "LI" && content === "Marketing"
    );
    const teachingTab = screen.getByText(
      (content, element) =>
        element.tagName === "LI" && content === "Teaching & Academics"
    );

    // Initially, "Development" content is displayed
    expect(screen.getByText("WordPress")).toBeInTheDocument();

    // Click on "Finance & Accounting" tab
    fireEvent.click(accountingTab);
    expect(screen.getByText("Accounting")).toBeInTheDocument();
    expect(screen.queryByText("WordPress")).not.toBeInTheDocument();

    // Click on "Design" tab
    fireEvent.click(designTab);
    expect(screen.getByText("Adobe Photoshop")).toBeInTheDocument();
    expect(screen.queryByText("Accounting")).not.toBeInTheDocument();

    // Click on "Marketing" tab
    fireEvent.click(marketingTab);
    expect(screen.getByText("Googgle Ads (Adwords)")).toBeInTheDocument();
    expect(screen.queryByText("Adobe Photoshop")).not.toBeInTheDocument();

    // Click on "Teaching & Academics" tab
    fireEvent.click(teachingTab);
    expect(screen.getByText("Math")).toBeInTheDocument();
    expect(screen.queryByText("Googgle Ads (Adwords)")).not.toBeInTheDocument();
  });
});
