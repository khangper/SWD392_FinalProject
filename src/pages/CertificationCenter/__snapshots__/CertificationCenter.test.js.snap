// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CertificationCenter renders the CertificationCenter component 1`] = `
<div>
  <div
    class="CertificationCenterPage"
  >
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <div
      class="CC_215certibg"
    >
      <div
        class="CC-grid-container"
      >
        <div
          class="CC-cert_banner_text CC-grid-item"
        >
          <h1>
            Certification Center
          </h1>
          <p>
            For Students and Instructors
          </p>
          <ul
            class="CC-certi_icons"
          >
            <li>
              <a
                class="CC-edttslogo"
                href="#"
              >
                <img
                  alt=""
                  src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
                />
              </a>
            </li>
            <li>
              <div
                class="CC-edttsplus"
              >
                <img
                  alt=""
                  src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/plus.svg"
                />
              </div>
            </li>
            <li>
              <a
                class="CC-edttslogo1"
                href="#"
              >
                <img
                  alt=""
                  src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/certicon.svg"
                />
              </a>
            </li>
          </ul>
          <a
            href="/certificationfillform"
          >
            <button
              class="CC-certi-btn"
            >
              Start Certification
            </button>
          </a>
        </div>
      </div>
    </div>
    <div
      class="CC_215xd5"
    >
      <div
        class="CC-grid-container"
      >
        <div
          class="CC-title889"
        >
          <h2>
            Find Certificate
          </h2>
        </div>
      </div>
      <div
        class="CC1-grid-container"
      >
        <div
          class="CC1-grid-item"
        >
          <div
            class="ui search focus"
          >
            <div
              class="ui left icon input swdh11 swdh19"
            >
              <input
                class="prompt srch_explore"
                placeholder="# Number"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div
          class="CC1-grid-item"
        >
          <div
            class="ui search focus"
          >
            <div
              class="ui left icon input swdh11 swdh19"
            >
              <input
                class="prompt srch_explore"
                placeholder="Full Name"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div
          class="CC1-grid-item"
        >
          <div
            class="ui search focus"
          >
            <select
              class="ui hj145 dropdown cntry152 prompt srch_explore"
            >
              <option
                value=""
              >
                Select Category
              </option>
              <option
                value="1"
              >
                Development
              </option>
              <option
                value="2"
              >
                Finance & Accounting
              </option>
              <option
                value="3"
              >
                Design
              </option>
              <option
                value="4"
              >
                Marketing
              </option>
              <option
                value="5"
              >
                Teaching & Academics
              </option>
            </select>
          </div>
        </div>
        <div
          class="CC1-grid-item"
        >
          <div
            class="ui search focus"
          >
            <button
              class="CC-login-btn"
              type="submit"
            >
              Find Certificate
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="CC_215xd5"
    >
      <div
        class="CC-grid-container"
      >
        <div
          class="CC-title589 "
        >
          <h2>
            Our Certification
          </h2>
          <p>
            We prepared tests for the most popular categories and get cerificate
          </p>
        </div>
      </div>
      <div
        class="CC-grid-container"
      >
        <div
          class="CC-container"
        >
          <ul
            class="CC-tabs"
          >
            <li
              class="active"
            >
              Development
            </li>
            <li
              class=""
            >
              Finance & Accounting
            </li>
            <li
              class=""
            >
              Design
            </li>
            <li
              class=""
            >
              Marketing
            </li>
            <li
              class=""
            >
              Teaching & Academics
            </li>
          </ul>
          <div
            class="tab-content"
          >
            <div
              class="CC-grid-container"
            >
              <div
                class="CC-development"
              >
                <div
                  class="CC-certificates"
                >
                  <div
                    class="CC3-grid-container"
                  >
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      WordPress
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      HTML CSS
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      MotoCMS 3
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      Joomla
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      OpenCart
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      Joomla Pro
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      WordPress Pro
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      WordPress Elementor
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      WordPress Elementor Pro
                    </a>
                    <a
                      class="CC-certilink__152"
                      href="#"
                    >
                      PrestaShop
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="CC_485td5"
    >
      <div
        class="CC-grid-container"
      >
        <div
          class="CC-titleceti89"
        >
          <h2>
            Who Can Get Benefit From This?
          </h2>
        </div>
      </div>
      <div
        class="CC4-grid-container"
      >
        <div
          class="CC4-grid-item"
        >
          <div
            class="who_get"
          >
            <div
              class="who_img"
            >
              <img
                alt="Student"
                src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/student.svg"
              />
            </div>
            <h4>
              Students
            </h4>
          </div>
        </div>
        <div
          class="CC4-grid-item"
        >
          <div
            class="who_get"
          >
            <div
              class="who_img"
            >
              <img
                alt="Instructor"
                src="https://gambolthemes.net/html-items/cursus-new-demo/images/cerificate_center/instructor.svg"
              />
            </div>
            <h4>
              Instructor
            </h4>
          </div>
        </div>
      </div>
    </div>
    <div
      class="CC_215xd5"
    >
      <div
        class="CC-grid-container"
      >
        <div
          class="CC-title589"
        >
          <h2>
            What Will You Get?
          </h2>
          <p>
            Cursus company, which confirms your skills and knowledge of Certification
          </p>
        </div>
      </div>
      <div
        class="CC5-grid-container"
      >
        <div
          class="CC-knowledge_dts"
        >
          <p>
            Morbi eget elit eget turpis varius mollis eget vel massa. Donec porttitor, sapien eget commodo vulputate, erat felis aliquam dolor, non condimentum libero dolor vel ipsum. Sed porttitor nisi eget nulla ullamcorper eleifend. Fusce tristique sapien nisi, vel feugiat neque luctus sit amet. Quisque consequat quis turpis in mattis. Maecenas eget mollis nisl. Cras porta dapibus est, quis malesuada ex iaculis at. Vestibulum egestas tortor in urna tempor, in fermentum lectus bibendum. In leo leo, bibendum at pharetra at, tincidunt in nulla. In vel malesuada nulla, sed tincidunt neque. Phasellus at massa vel sem aliquet sodales non in magna. Ut tempus ipsum sagittis neque cursus euismod. Vivamus luctus elementum tortor, ac aliquet dolor vehicula et. Nulla vehicula pharetra lacus ornare gravida. Vivamus mollis ullamcorper dui quis gravida. Aenean pulvinar pulvinar arcu a suscipit.
          </p>
          <button
            class="CC-knowledge_btn"
          >
            Knowledge Base
          </button>
        </div>
      </div>
    </div>
    <div
      class="CC6-grid-container"
    >
      <footer
        class="footer footer-large-container"
      >
        <div
          class="footer-links"
        >
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/about-us"
            >
              About
            </a>
            <a
              class="link"
              href="/blog-single-view/:id"
            >
              Blog
            </a>
            <a
              class="link"
              href="/career"
            >
              Careers
            </a>
            <a
              class="link"
              href="/press"
            >
              Press
            </a>
          </div>
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/help"
            >
              Help
            </a>
            <a
              class="link"
              href="/coming-soon"
            >
              Advertise
            </a>
            <a
              class="link"
              href="/coming-soon"
            >
              Developers
            </a>
            <a
              class="link"
              href="/contactus"
            >
              Contact Us
            </a>
          </div>
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/termsofuse"
            >
              Copyright Policy
            </a>
            <a
              class="link"
              href="/termsofuse"
            >
              Terms
            </a>
            <a
              class="link"
              href="/termsofuse"
            >
              Privacy Policy
            </a>
            <a
              class="link"
              href="/sitemap"
            >
              Sitemap
            </a>
          </div>
          <div
            class="footer-column"
          >
            <button
              class="teach-button"
            >
              Teach on Cursus
            </button>
            <div
              class="language-selector"
            >
              <button>
                Language 
                <span>
                  ▼
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          class="footer-bottom"
        >
          <div
            class="footer-copy"
          >
            <img
              alt=""
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
            />
            © 2024 
            <strong>
              Cursus
            </strong>
            . All Rights Reserved.
          </div>
          <div
            class="footer-social"
          >
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="facebook.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="x.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="google+.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="linkedin.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="insta.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="pinterest.png"
              />
            </a>
          </div>
        </div>
      </footer>
    </div>
  </div>
</div>
`;
